src/cassio/table/base_table.py

```diff
@@ -612,10 +607,12 @@ async def aexecute_cql(
             statement = SimpleStatement(final_cql)
             logger.debug(f'aExecuting statement "{final_cql}" as simple (unprepared)')
         else:
-            statement = self._obtain_prepared_statement(final_cql)
+            statement = await asyncio.to_thread(
```

Comment:
It seems there's no possibility to prepare a statement async in the driver atm.
So delegating to a thread...